src/api.js

```diff
@@ -2,15 +2,22 @@ import { examRequiresAccessToken, store } from './data';
 
 export function isExam() {
   const { exam } = store.getState().examState;
-  return exam.id !== null;
+  return !!exam?.id;
 }
 
 export function getExamAccess() {
```

Comment:
Note: I removed the logError from both getExamAccess and fetchExamAccess. As we'd discussed, it's not really needed since it's not exactly an error if a unit is not an exam, so we probably don't want extraneous logs.